README.rst

```diff
@@ -0,0 +1,99 @@
+Before the start, make sure that you have an account on `PyPI`_
+
+
+Quickstart
+----------
+ 
+1. Install Cookiecutter
+
+Create and activate an environment for the package project.
+
+.. code-block:: bash
+
+    conda create -n <repo_name> python=<python_version>
```

Comment:
Simpler to install cookiecutter here too:

```
conda create -n <repo_name> python=<python_version> cookiecutter
```

And might as well include the activation instructions here.

Reply:
Section `Install cookiecutter` removed